sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.students.controller.ts

```diff
@@ -133,10 +137,34 @@ export class AssessmentStudentsController extends BaseController {
     @Param("applicationId", ParseIntPipe) applicationId: number,
     @UserToken() userToken: StudentUserToken,
   ): Promise<RequestAssessmentSummaryAPIOutDTO[]> {
-    return this.assessmentControllerService.getPendingAndDeniedAppeals(
-      applicationId,
-      userToken.studentId,
-    );
+    const pendingAndDeniedAppeals =
+      await this.assessmentControllerService.getPendingAndDeniedAppeals(
+        applicationId,
+        userToken.studentId,
+      );
+    const inProgressAndDeclinedApplicationOfferingChangeRequests =
+      await this.applicationOfferingChangeRequestService.getApplicationOfferingChangeRequestsByStatus(
+        applicationId,
+        userToken.studentId,
+        [
+          ApplicationOfferingChangeRequestStatus.InProgressWithStudent,
+          ApplicationOfferingChangeRequestStatus.InProgressWithSABC,
+          ApplicationOfferingChangeRequestStatus.DeclinedByStudent,
+          ApplicationOfferingChangeRequestStatus.DeclinedBySABC,
+        ],
+      );
+    const studentInProgressAndDeclinedApplicationOfferingChangeRequestsArray: RequestAssessmentSummaryAPIOutDTO[] =
+      inProgressAndDeclinedApplicationOfferingChangeRequests.map((request) => ({
+        id: request.id,
+        submittedDate: request.createdAt,
+        status: request.applicationOfferingChangeRequestStatus,
+        requestType:
+          RequestAssessmentTypeAPIOutDTO.ApplicationOfferingChangeRequest,
+      }));
+    return [
+      ...pendingAndDeniedAppeals,
+      ...studentInProgressAndDeclinedApplicationOfferingChangeRequestsArray,
```

Comment:
I believe, we don't use the object type name in the variable name, unless it's needed